app/display/linearmeter/src/main/java/org/csstudio/display/extra/widgets/linearmeter/RTLinearMeter.java

```diff
@@ -735,7 +735,7 @@ public void setShowWarnings(boolean showWarnings) {
     private boolean isValueWaitingToBeDrawn = false;
     private double valueWaitingToBeDrawn = Double.NaN;
     /** @param newValue Current value */
-    public void setCurrentValue(double newValue)
+    public void setCurrentValue(double newValue, boolean forceRedraw)
```

Comment:
I'm not a fan of boolean flags, I would write an enum

enum DrawAction {
   ForceRedraw
   Draw
   NoRender
}

or something like this.